### DISCUSSION TITLE
Breadcrumb tooltip in link dialog

### DISCUSSION THREAD

**Discussion by thumDer** [outline/outline#5231] [Ideas]
The link dialog could show the full breadcrumb when the user hovers over it if it is truncated by ellipses. This would be very useful in deeply structured collections.

![kép](https://user-images.githubusercontent.com/54556708/233323988-3b3617e2-b769-4fcc-9a7a-7561319f93b1.png)

**Comment by bvandevliet** [comment]
I second this. Also when creating a backlink and if there are multiple docs with the same name but in different locations, it's now a blind guess picking the right one.

**Reply by tommoor** [reply]
This was added in https://github.com/outline/outline/releases/tag/v0.84.0
